packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+
+                <!-- TODO: Translate "Next" -->
+                <sp-action-button
+                    quiet
+                    size="s"
+                    aria-label="Next"
+                    title="Next"
+                    class="spectrum-ActionButton spectrum-Calendar-nextMonth"
+                    ?disabled=${this.disabled}
+                    @click=${this.handleNextMonth}
+                >
+                    <div slot="icon">
+                        <slot name="prev-icon">
+                            <sp-icon-chevron-right></sp-icon-chevron-right>
+                        </slot>
+                    </div>
+                </sp-action-button>
+            </div>
+        `;
+    }
+
+    public renderCalendarGrid(): TemplateResult {
+        return html`
+            <!-- TODO: Implement keyboard navigation -->
+            <div
+                class="spectrum-Calendar-body"
+                role="grid"
+                tabindex=${ifDefined(!this.disabled ? '0' : undefined)}
+                aria-readonly="true"
+                aria-disabled=${this.disabled}
+            >
+                <table role="presentation" class="spectrum-Calendar-table">
+                    ${this.renderCalendarTableHead()}
+                    ${this.renderCalendarTableBody()}
+                </table>
+            </div>
+        `;
+    }
+
+    public renderCalendarTableHead(): TemplateResult {
+        return html`
+            <thead role="presentation">
+                <tr role="row">
+                    ${this._getWeekdays().map((weekday) =>
+                        this.renderWeekdayColumn(weekday)
+                    )}
+                </tr>
+            </thead>
+        `;
+    }
+
+    public renderWeekdayColumn(weekday: CalendarWeekday): TemplateResult {
+        return html`
+            <th
+                role="columnheader"
+                scope="col"
+                class="spectrum-Calendar-tableCell"
+            >
+                <abbr class="spectrum-Calendar-dayOfWeek" title=${weekday.long}>
+                    ${weekday.narrow}
+                </abbr>
+            </th>
+        `;
+    }
+
+    public renderCalendarTableBody(): TemplateResult {
+        return html`
+            <tbody role="presentation">
+                ${[...new Array(this._getWeeksInCurrentMonth()).keys()].map(
+                    (weekIndex) => this.renderCalendarTableRow(weekIndex)
+                )}
+            </tbody>
+        `;
+    }
+
+    public renderCalendarTableRow(weekIndex: number): TemplateResult {
+        return html`
+            <tr role="row">
+                ${this._getDatesInWeek(weekIndex).map((calendarDate) =>
+                    this.renderCalendarTableCell(calendarDate)
+                )}
+            </tr>
+        `;
+    }
+
+    public renderCalendarTableCell(calendarDate: CalendarDate): TemplateResult {
+        const isOutsideMonth = calendarDate.month !== this._currentDate.month;
```

Comment:
Can you specify  the type of isOutsideMonth here as Boolean?

Reply:
Type boolean is inferred, ESLint would tell me to remove the type annotation
![image](https://github.com/adobe/spectrum-web-components/assets/4804498/c3795354-ebab-4808-b38d-44be7117f729)